ext/drmgr/drmgr.h

```diff
@@ -452,6 +465,53 @@ drmgr_unregister_bb_instrumentation_ex_event(drmgr_app2app_ex_cb_t app2app_func,
                                              drmgr_insertion_cb_t insertion_func,
                                              drmgr_ilist_ex_cb_t instru2instru_func);
 
+DR_EXPORT
+/**
+ * Registers callback functions for the third
+ * instrumentation stage: instrumentation
+ * insertion.  drmgr will call \p func for each instruction with the
+ * specific opcode \p opcode.
+ *
+ * More than one call-back function can be mapped to the same opcode. Their
+ * execution sequence is determined by their priority \p priority if set. Ordering
+ * based on priority is also taken in to account with respect to insert bb events.
+ *
+ * Since this call-back is triggered during insert instrumentation,
```

Comment:
Hmm, try `git grep callback` and `git grep call-back`.  Certainly the former is used everywhere  else (except drbbdup).

Reply:
Done - I will fix drbbdup later.